database/migrations/2017_03_17_170318_convert_text_to_longtext.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Database\Migrations\Migration;
+
+class ConvertTextToLongtext extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::table('questions', function (Blueprint $table) {
+            //
+            DB::statement('ALTER TABLE questions MODIFY  blocks  LONGTEXT;');
+            DB::statement('ALTER TABLE questions MODIFY  description LONGTEXT;');
+            DB::statement('ALTER TABLE questions MODIFY  hint_text LONGTEXT;');
+        });
+    }
+
+    /**
+     * Reverse the migrations.
+     *
+     * @return void
+     */
+    public function down()
+    {
+        Schema::table('questions', function (Blueprint $table) {
+            //
+        });
```

Comment:
I have written this code but I think I mistakenly  discarded  the changes of this file :)